src/main/java/com/znsio/teswiz/runner/Driver.java

```diff
@@ -603,4 +606,15 @@ public void doubleTap(WebElement element) {
                 .addAction(touch.createPointerUp(PointerInput.MouseButton.LEFT.asArg()));
         appiumDriver.perform(Arrays.asList(clickPosition));
     }
+
+    public void horizontalSwipeWithGesturesPlugin(WebElement element, String direction) {
+        RemoteWebElement remoteWebElement = (RemoteWebElement) element;
+        if (direction.equalsIgnoreCase("left") || direction.equalsIgnoreCase("right")) {
```

Comment:
you can use the Direction enum instead of hardcoded values

Reply:
updated with the direction value from Direction class